src/apps/backend/modules/task/internal/task-reader.ts

```diff
@@ -25,14 +25,14 @@ export default class TaskReader {
     return TaskUtil.convertTaskDBToTask(taskDb);
   }
 
-  public static async getTaskByNameForAccount(params: GetTaskByNameParams): Promise<Task> {
+  public static async getTaskByTitleForAccount(params: GetTaskByTitleParams): Promise<Task> {
```

Comment:
This does not sound right, task title cannot guaranteed be unique. For example, I can add a daily task to work out.